source/tutorials/fmt.rst

```diff
@@ -83,13 +83,27 @@ Here the quality seems relatively low in the first few bases, and then seems to
      --p-trunc-len 150 \
      --i-demultiplexed-seqs fmt-tutorial-demux-1.qza \
      --o-representative-sequences rep-seqs-1.qza \
-     --o-table table-1.qza
+     --o-table table-1.qza \
+     --o-denoising-stats stats-1.qza
    qiime dada2 denoise-single \
      --p-trim-left 13 \
      --p-trunc-len 150 \
      --i-demultiplexed-seqs fmt-tutorial-demux-2.qza \
      --o-representative-sequences rep-seqs-2.qza \
-     --o-table table-2.qza
+     --o-table table-2.qza \
+     --o-denoising-stats stats-2.qza
+
+Viewing denoising stats
+-----------------------
+
+The ``denoise-single`` commands return basic statistics about the denoising process, and can be visualized with the following:
+
+.. command-block::
+
+   qiime metadata tabulate \
+     --m-input-file stats-1.qza \
+     --m-input-file stats-2.qza \
```

Comment:
Does this work? The columns will be the same name, but the samples will be different. I don't remember if metadata merging can handle unrelated samples with the same columns

Reply:
Good call - it almost certainly doesn't work!

```bash
qiime metadata tabulate \
  --m-input-file stats-1.qza \
  --m-input-file stats-2.qza \
  --o-visualization whee.qzv
Traceback (most recent call last):
  File "/Users/matthew/.conda/envs/q2dev/bin/qiime", line 11, in <module>
    sys.exit(qiime())
  File "/Users/matthew/.conda/envs/q2dev/lib/python3.5/site-packages/click/core.py", line 722, in __call__
    return self.main(*args, **kwargs)
  File "/Users/matthew/.conda/envs/q2dev/lib/python3.5/site-packages/click/core.py", line 697, in main
    rv = self.invoke(ctx)
  File "/Users/matthew/.conda/envs/q2dev/lib/python3.5/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/matthew/.conda/envs/q2dev/lib/python3.5/site-packages/click/core.py", line 1066, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/Users/matthew/.conda/envs/q2dev/lib/python3.5/site-packages/click/core.py", line 895, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/Users/matthew/.conda/envs/q2dev/lib/python3.5/site-packages/click/core.py", line 535, in invoke
    return callback(*args, **kwargs)
  File "/Users/matthew/.conda/envs/q2dev/lib/python3.5/site-packages/q2cli/commands.py", line 244, in __call__
    arguments, missing_in, verbose, quiet = self.handle_in_params(kwargs)
  File "/Users/matthew/.conda/envs/q2dev/lib/python3.5/site-packages/q2cli/commands.py", line 323, in handle_in_params
    verbose, kwargs, fallback=cmd_fallback)
  File "/Users/matthew/.conda/envs/q2dev/lib/python3.5/site-packages/q2cli/handlers.py", line 492, in get_value
    return metadata[0].merge(*metadata[1:])
  File "/Users/matthew/src/qiime2/qiime2/qiime2/metadata/metadata.py", line 683, in merge
    ', '.join([repr(e) for e in columns.get_duplicates()]))
ValueError: Cannot merge metadata with overlapping columns. The following columns overlap: 'denoised', 'filtered', 'input', 'non-chimeric'
(q2dev)
~/data/fmt
```